### ISSUE TITLE
H.264 video over rtp sometimes not playing for iOS

### ISSUE THREAD

**Issue by reeonce** [Bilibili/ijkplayer#994]
##### My use case: 
There is another device that sends the rtp video stream to my device in my intranet. My app needs to display that video.

##### My implementation:
Store the sdp file to my device via `NSFileManager`. The content is:
```
    m=video 3069 RTP/AVP 96
    a=rtpmap:96 H264/90000
    a=framerate:25
    c=IN IP4 127.0.0.1
```
then, create a player and play:
```
self.player = [[IJKFFMoviePlayerController alloc] initWithContentURL:self.sdpURL withOptions:options];
[self.player prepareToPlay];
```

I can always get the video be played with `ffplay -i sdp.sdp`

I can sometimes get the video played on my iPhone, but never in my iOS simulator.

##### logs:
```
sdp content: 
m=video 3069 RTP/AVP 96
a=rtpmap:96 H264/90000
a=framerate:25
c=IN IP4 127.0.0.1

[;===== custom modules begin =====
register protocol: async
register protocol: ijkhttphook
register protocol: ijkinject
register protocol: ijklongurl
register protocol: ijksegment
register protocol: ijktcphook
register demuxer : ijklivehook
===== custom modules end =====
av_version_info: ff2.8--ijk0.4.4.1--dev0.3.3--rc7
ijkmp_set_inject_opaque(0x7fa4fd6c7940)
ijkmp_set_inject_opaque()=void
2016-03-07 16:53:01.146 xxx[90495:6114062] OK setup GL
ijkmp_ios_set_view(glView=0x7fa4fd709e90)
ijkmp_ios_set_view(glView=0x7fa4fd709e90)=void
2016-03-07 16:53:01.148 xxx[90495:6114062] invalidateRenderBuffer
2016-03-07 16:53:01.148 xxx[90495:6114292] IJKSDLGLView: setupDisplay not ready
ijkmp_set_data_source(url="/Users/xxxxx/Library/Developer/CoreSimulator/Devices/83184200-690A-4073-8B0D-0BC14EB24794/data/Containers/Data/Application/B1FEF2D1-C334-4AFB-812A-97525E3C7168/Library/Documentation/sdp.sdp")
ijkmp_set_data_source(url="/Users/xxxxx/Library/Developer/CoreSimulator/Devices/83184200-690A-4073-8B0D-0BC14EB24794/data/Containers/Data/Application/B1FEF2D1-C334-4AFB-812A-97525E3C7168/Library/Documentation/sdp.sdp")=0
ijkmp_prepare_async()
===== versions =====
FFmpeg       : ff2.8--ijk0.4.4.1--dev0.3.3--rc7
libavutil    : 54.31.100
libavcodec   : 56.60.100
libavformat  : 56.40.101
libswscale   : 3.1.101
libswresample: 1.2.101
===== options =====
player-opts : video-pictq-size             = 3
player-opts : overlay-format               = fcc-_es2
player-opts : max-fps                      = 30
player-opts : framedrop                    = 0
player-opts : videotoolbox-max-frame-width = 960
player-opts : videotoolbox                 = 0
player-opts : start-on-prepared            = 1
format-opts : ijkinject-opaque             = 140346603108672
format-opts : user-agent                   = ijkplayer
format-opts : auto_convert                 = 0
format-opts : timeout                      = 30000000
format-opts : reconnect                    = 1
format-opts : safe                         = 0
===================
ijkmp_prepare_async()=0
2016-03-07 16:53:04.774 xxx[90495:6114062] invalidateRenderBuffer
2016-03-07 16:53:04.774 xxx[90495:6114279] IJKSDLGLView: setupDisplay not ready
Statistics: 81 bytes read, 0 seeks
/Users/xxxxx/Library/Developer/CoreSimulator/Devices/83184200-690A-4073-8B0D-0BC14EB24794/data/Containers/Data/Application/B1FEF2D1-C334-4AFB-812A-97525E3C7168/Library/Documentation/sdp.sdp: Invalid data found when processing input
2016-03-07 16:53:04.782 xxx[90495:6114062] FFP_MSG_ERROR: 0
[fg127,127,127;2016-03-07 16:53:04:783 xxx[90495:6114062] IJKMPMoviePlayBackStateDidChange 2: paused
```

**Comment by reeonce** [comment]
Found the reason, it is cause by the wrong ffmpeg config.

Actually, I compiled ffmpeg with the `module-default.sh` and run on my device, it worked. In order to minimise the ffmpeg size, I recompiled it with the `module-lite-hevc.sh` and tried to enabled rtp by
```
export COMMON_FF_CFG_FLAGS="$COMMON_FF_CFG_FLAGS --enable-protocol=rtp"
export COMMON_FF_CFG_FLAGS="$COMMON_FF_CFG_FLAGS --enable-protocol=sctp"
export COMMON_FF_CFG_FLAGS="$COMMON_FF_CFG_FLAGS --enable-protocol=srtp"
```
But this didn't work, which make the simulator never got the chance to be worked. However, for my iPhone, which has cached the previous framework, the player worked by chance. After I removing the app and re-install it, it changed to be problematic.

**So, the real problem is, how can I make it working with the minimal ffmpeg size?**, I am not familiar with ffmpeg. 

Thanks.
